### PR TITLE
add unstable book to the bookshelf

### PR DESCRIPTION
**Pull request by steveklabnik**
r? @frewsxcv @GuillaumeGomez

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

File: src/doc/index.md [hunk 1]
```diff
@@ -42,6 +43,7 @@ landed before then. That work is being tracked [here][38643].
 [ref]: reference/index.html
 [38643]: https://github.com/rust-lang/rust/issues/38643
 [err]: error-index.html
-[book]: book/index.html
+[unstable-book]: book/index.html
```
Comment (durka) [inline src/doc/index.md]: This doesn't look right...

Comment (steveklabnik) [inline src/doc/index.md]: sigh, thanks

**Comment by steveklabnik** [comment]
fixed up, ready for another review

**Comment by steveklabnik** [comment]
Why does bors think there's merge conflicts?

**Pull request merged**